a10_octavia/tests/common/a10constants.py

```diff
@@ -26,3 +26,7 @@
 MOCK_TEMPLATE_NAME = 'mock-template-name-1'
 MOCK_CERT_ACTION = 'import'
 MOCK_CERT_TYPE = 'pem'
+
+SERVICE_GROUP_CONF_SECTION = 'service_group'
+POOL_LB_ALGO_SOURCE_IP = 'SOURCE_IP'
```

Comment:
Import this instead
https://github.com/openstack/octavia/blob/49036845c118aec11740f18680b779f99113bcfc/octavia/common/constants.py#L120